modules/darwin/src/main/ts/ephox/darwin/mouse/MouseSelection.ts

```diff
@@ -42,13 +39,14 @@ export default function (bridge: WindowBridge, container: Element, isRoot: (e: E
   };
 
   /* Keep this as lightweight as possible when we're not in a table selection, it runs constantly */
-  const mouseup = function (_event?: EventArgs) {
-    cursor.each(clearState);
+  const mouseup = (_event?: EventArgs) => {
+    clearstate();
   };
 
   return {
+    clearstate,
```

Comment:
This requires a minor version bump to darwin, so can you please add those details to `versions.txt` please.